test-runner.sh

```diff
@@ -0,0 +1,65 @@
+#!/usr/bin/env bash
+
+# Build a framework, package, upload it, and then run its integration tests.
+# (Or all frameworks depending on arguments.) Expected to be called by test.sh
+
+# Exit immediately on errors
+set -e
+
+REPO_ROOT_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+
+if [ "$FRAMEWORK" = "all" ]; then
+    # randomize the FRAMEWORK_LIST
```

Comment:
This was only desired for flushing out staleness in CI.  I think the randomization can die now, if you're willing to change that.